enki/PhysicalEngine.cpp

```diff
@@ -225,14 +225,15 @@ namespace Enki
 	
 	const double PhysicalObject::g = 9.81;
 	
-	PhysicalObject::PhysicalObject(void) :
+	PhysicalObject::PhysicalObject(bool mouvable) :
```

Comment:
Is this necessary? A priori I would imagine that objects with non-infinite mass can be moved. Do you see a specific use case for that?

Btw, it should be _movable_ rather than _mouvable_